rules/falco-deprecated_rules.yaml

```diff
@@ -110,68 +134,93 @@
   items: [google.com, www.yahoo.com]
 
 - rule: Unexpected outbound connection destination
-  desc: > 
-    Detect any outbound connection to a destination outside of an allowed set of ips, networks, or domain names. 
-    This rule absolutely requires profiling your environment beforehand. Network-based rules are extremely crucial 
-    in any security program, as they can often provide the only definitive evidence. However, effectively operationalizing 
-    them can be challenging due to the potential for noise.
+  desc: >
+    Detect any outbound connection to a destination outside of an allowed
+    set of ips, networks, or domain names. This rule absolutely requires
+    profiling your environment beforehand. Network-based rules are extremely
+    crucial in any security program, as they can often provide the only
+    definitive evidence. However, effectively operationalizing them can be
+    challenging due to the potential for noise.
   condition: >
-    outbound 
+    outbound
     and not ((fd.sip in (allowed_outbound_destination_ipaddrs)) or
-             (fd.snet in (allowed_outbound_destination_networks)) or
-             (fd.sip.name in (allowed_outbound_destination_domains)))
+            (fd.snet in (allowed_outbound_destination_networks)) or
+            (fd.sip.name in (allowed_outbound_destination_domains)))
   enabled: false
-  output: Disallowed outbound connection destination (connection=%fd.name lport=%fd.lport rport=%fd.rport fd_type=%fd.type fd_proto=fd.l4proto evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty %container.info)
+  output: >
+    Disallowed outbound connection destination
+    (connection=%fd.name lport=%fd.lport rport=%fd.rport fd_type=%fd.type
+    fd_proto=fd.l4proto evt_type=%evt.type user=%user.name user_uid=%user.uid
+    user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath
+    parent=%proc.pname command=%proc.cmdline terminal=%proc.tty
+    %container.info)
+
   priority: NOTICE
-  tags: [maturity_deprecated, host, container, network, mitre_command_and_control, TA0011]
+  tags: [maturity_deprecated, host, container, network,
```

Comment:
Tags now seem formatted inconsistently, for example few lines above we have:

```
tags: [
    maturity_deprecated, host, container, network,
    mitre_latera_movement, T1021.004
  ]
```

Reply:
Hey @incertum tags format is updated as per your suggestion.